README.md

```diff
@@ -61,3 +61,26 @@ aws s3 cp s3://bucket/foo.fastq.zstd | \
     siz2fastq -z >(aws s3 cp - s3://bucket/foo_1.fastq.gz) \
                  >(aws s3 cp - s3://bucket/foo_2.fastq.gz) \
 ```
+
+## SIZer
+
+Convert paired short-read FASTQ files to split, interleaved, zstd-compressed (SIZ) format.
+Typically used within a data processing pipeline and with compress-upload.sh.
+
+Presently, input FASTQ files with hard wrapping are not supported.
+```
+Usage: ./SIZer [-p <local_prefix>] <reads_per_chunk> <r1_fastq> <r2_fastq> <s3_output_prefix> <compress_script_path> <max_concurrent_jobs> <zstd_threads>
+Options:
+  -p <local_prefix>  Prefix for temporary chunk files (default: work_chunk)
+Arguments:
+  reads_per_chunk      Max read pairs per output chunk
+  r1_fastq             Path to R1 FASTQ file (can be .gz)
+  r2_fastq             Path to R2 FASTQ file (can be .gz)
+  s3_output_prefix     S3 prefix for output files
+  compress_script_path Path to compression/upload script
+  max_concurrent_jobs  Max simultaneous background jobs
+  zstd_threads         Number of zstd threads for script
```

Comment:
This is a lot of positional arguments, and some of them seem like they could have good defaults.  What would you think of:

```
    ./SIZer [-p <local_prefix>] [-c <reads_per_chunk>] [-j <max_concurrent_jobs>] [-t <zstd_threads>] <r1_fastq> <r2_fastq> <s3_output_prefix> <compress_script_path>
```

Where `reads_per_chunk` defaults to 1M, `max_concurrent_jobs` defaults to number of cores / 4, and `zstd_threads` defaults to 4. (Or possibly some number other than 4; you probably have a better sense of the right balance.)